src/EnergyPlus/Autosizing/CoolingCapacitySizing.cc

```diff
@@ -167,6 +166,7 @@ Real64 CoolingCapacitySizer::size(EnergyPlusData &state, Real64 _originalValue,
                             if (DDNum > 0 && TimeStepNumAtMax > 0) {
                                 OutTemp = state.dataSize->DesDayWeath(DDNum).Temp(TimeStepNumAtMax);
                             }
+                            if (this->dataCoolCoilType > -1) OutTemp = GetCoilSourceTempUsedForSizing(this->dataCoolCoilType, OutTemp);
```

Comment:
Protected call to new function that mimics GetVSCoilRatedSourceTemp added 7 days ago. If a parent object (or unit test) does not set the coil type, notihng happens, and OutTemp remains unchanged.